### PR TITLE
X874 solution transfer

### PR DESCRIPTION
**Pull request by seenanair**
New feature to record operation of Solution transfer of samples

### CODE REVIEW

File: cypress/fixtures/example.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (khelwood) [inline cypress/fixtures/example.json:1]: Is this file actually needed?

Comment (seenanair) [reply cypress/fixtures/example.json]: No, it's not needed. Deleted 👍

**Review by khelwood — APPROVED** [review]
